docs/tasks/administer-cluster/reconfigure-kubelet.md

```diff
@@ -0,0 +1,264 @@
+
+### Workaround: Add system:Nodes subject to the system:Node ClusterRoleBinding 
+
+The Node authorizer does not yet pay attention to which ConfigMaps are assigned to which Nodes. If you currently use the Node authorizer, your Kubelets will not be automatically granted permission to download their respective ConfigMaps. The temporary workaround used in this document is to add the system:Nodes subject to the system:Node ClusterRoleBinding, which will grant the required permissions. The Node authorizer will be extended before the Dynamic Kubelet Configuration feature graduates from alpha, so doing this in production should never be necessary.
+
+Edit the `system:Node` ClusterRoleBinding with `kubectl edit clusterrolebinding system:Node` and add the following subject under `subjects`:
+
+```
+subjects:                                                                       
+- apiGroup: rbac.authorization.k8s.io                                           
+  kind: Group                                                                   
+  name: system:Nodes 
+```
+
+
+### Generating a file that contains the current configuration
+
+The Dynamic Kubelet Configuration feature allows you to provide an override for the entire configuration object, rather than a per-field overlay. This is a simpler model that makes it easier to trace the source of configuration values and debug issues. The compromise, however, is that you must start with knowledge of the existing configuration to ensure that you only change the fields you intend to change.
+
+In the future, the Kubelet will be bootstrapped from a file on disk, and you will simply edit a copy of this file (which, as a best practice, should live in version control) while creating the first Kubelet ConfigMap. Today, however, the Kubelet is still bootstrapped with command-line flags. Fortunately, there is a dirty trick you can use to generate a config file containing a Node's current configuration. The trick involves hitting the Kubelet server's `configz` endpoint via the kubectl proxy. This endpoint, in its current implementation, is intended to be used only as a debugging aid, which is part of why this is a dirty trick. There is ongoing work to improve the endpoint, and in the future this will be a less "dirty" operation. This trick also requires the `jq` command to be installed on your machine, for unpacking and editing the json response from the endpoint.
+
+Do the following to generate the file:
+
+1. Pick a Node to reconfigure. We will refer to this Node's name as NODE_NAME.
+2. Start the kubectl proxy in the background with `kubectl proxy --port=8001 &`
+3. Run the following command to download and unpack the configuration from the configz endpoint: 
+
+```
+curl http://localhost:8001/api/v1/proxy/Nodes/NODE_NAME/configz | jq '.kubeletconfig|.kind="KubeletConfiguration"|.apiVersion="kubeletconfig/v1alpha1"' > kubelet
```

Comment:
nit: Nodes -> nodes, `curl` -> `curl -sSL`, `NODE_NAME` -> `${NODE_NAME}`
call the file `kubelet_configz_${NODE_NAME}`